.github/workflows/boostrap-workflow.yml

```diff
@@ -0,0 +1,23 @@
+name: bootstrap workflow
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 2
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: run boostrap.sh
+      run: |
+        cd ./
```

Comment:
Why cd is required?

Reply:
@lioramilbaum navigate to the root directory

Reply:
Navigation to the root directory is - `cd /` pay attention the dot is missing.
`cd ./` means navigate to the same directory you are already at